### DISCUSSION TITLE
Question: What does semi godemode do?

### DISCUSSION THREAD

**Discussion by LintStalker** [YimMenu/YimMenu#1627] [Q&A]
What does the semi godemode option do, and is it detectable by other people like godemode is?

**Comment by V1RU513N** [comment]
It increases the health speed regeneration

**Reply by LintStalker** [reply]
Cool!  Is it detectable like regular godmode is?

**Reply by Yimura** [reply]
YimMenu spoofs godmode so even that isn’t detected.

**Reply by LintStalker** [reply]
Cool!

**Comment by LintStalker** [comment]
This has been answered
